src/dunit/attributes.d

```diff
@@ -16,6 +18,40 @@ struct Tag
     string name;
 }
 
+struct EnabledIf
+{
+    bool condition;
```

Comment:
Only conditions that are evaluated at compile time are supported.
Can you support conditions that are evaluated at run time? Like that:

    @EnabledIf!(() => environment.get("foo") == "bar")

That's how it's done in https://github.com/funkwerk/boilerplate

Reply:
fixed